DEMOQA/cypress/tests/login.js

```diff
@@ -0,0 +1,204 @@
+            cy.get("#lastName").type(data.lastName + '$%^')
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(data.userNumber)
+
+            cy.get("#submit").click({ force: true })
+
+            cy.get("#firstName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+            cy.get("#lastName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+        })
+    })
+
+    it('cannot submit with invalid mobile ', () => {
+
+        testData.invalidNumbers.forEach((mobile, index) => {
+            cy.get("#firstName").type(testData.firstName)
+            cy.get("#lastName").type(testData.lastName)
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(mobile)
+
+            cy.get("#submit").click({ force: true })
+
+            cy.get("#userNumber").should('have.css', 'border-color', 'rgb(220, 53, 69)');
+        })
+    })
+
+    it('cannot submit with an invalid email', () => {
+        testData.invalidEmail.forEach((email, index) => {
+
+            cy.get("#firstName").type(testData.firstName)
+            cy.get("#lastName").type(testData.lastName)
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(testData.userNumber)
+            cy.get('#userEmail').type(email)
+
+            cy.get("#submit").click({ force: true })
+
+            cy.get("#userEmail").should('have.css', 'border-color', 'rgb(220, 53, 69)');
+        })
+    })
+
+    it('cannot submit with DOB with later than today', () => {
+
+        cy.fixture('testData').then((data) => {
+            cy.get("#firstName").type(data.firstName)
+            cy.get("#lastName").type(data.lastName)
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(data.userNumber)
+            cy.get('#dateOfBirthInput').click()
+            cy.get('.react-datepicker__month-select').select('January')
+            cy.get('.react-datepicker__year-select').select('2024')
+            cy.get('[role = "option"]').contains('15').click()
+
+            cy.get('#userEmail').type('vnphuong3001@')
+
+            cy.get("#submit").click({ force: true })
+            cy.get('#dateOfBirthInput').should('have.css', 'border-color', 'rgb(220, 53, 69)')
+        })
+    })
+
+    it('cannot submit with picture is not a image file', () => {
+
+        cy.fixture('testData').then((data) => {
+            cy.get("#firstName").type(data.firstName)
+            cy.get("#lastName").type(data.lastName)
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(data.userNumber)
+            cy.get('#uploadPicture').selectFile('cypress/fixtures/NAB_TC_studentRegistrationForm.xlsx')
+
+            cy.get("#submit").click({ force: true })
+            cy.get(".modal-content").should("not.be.visible");
+        })
+    })
+
+    it('cannot choose second gender', () => {
```

Comment:
This case we usually not implement in E2E test because this is not a flow, it is just a function of checkbox UI